supportedBackends/google/pipelines/batch/src/main/scala/cromwell/backend/google/pipelines/batch/runnable/RunnableBuilder.scala

```diff
@@ -45,14 +43,41 @@ object RunnableBuilder {
         )
     }
 
+    def withFlags(flags: List[RunnableFlag]): Runnable.Builder = {
+      flags.foldLeft(builder) {
+//        case (acc, RunnableFlag.Unspecified) =>
+        case (acc, RunnableFlag.IgnoreExitStatus) => acc // TODO: How do we handle this?
+        case (acc, RunnableFlag.RunInBackground) => acc.setBackground(true)
+        case (acc, RunnableFlag.AlwaysRun) => acc.setAlwaysRun(true)
+//        case (acc, RunnableFlag.PublishExposedPorts) =>
+//        case (acc, RunnableFlag.DisableImagePrefetch) =>
+      }
+    }
+
     def withVolumes(volumes: List[Volume]): Runnable.Builder = {
-      // TODO: Add mount options
-      val formattedVolumes = volumes.map(v => s"${v.getMountPath}:${v.getMountPath}")
+      val formattedVolumes = volumes.map { volume =>
+        val mountPath = volume.getMountPath
+        // TODO: Enable mount options once we know how to test different cases, for now, we keep this disabled
+        //       In theory, the commented code should be enough.
+//        val mountOptions = Option(volume.getMountOptionsList).map(_.asScala.toList).getOrElse(List.empty)
```

Comment:
In theory, this should be enough but I haven't tested multiple mount options, better to leave this commented until we verify that it works